packages/styled-components/rollup.config.js

```diff
@@ -28,7 +28,44 @@ const esm = {
 const getCJS = override => ({ ...cjs, ...override });
 const getESM = override => ({ ...esm, ...override });
 
+/**
+ * Support external dependencies on CommonJS modules without relying on named export inference
+ * (works similarly to more recent versions of Rollup's commonjs plugin)
```

Comment:
>  * (works similarly to more recent versions of Rollup's commonjs plugin)

Should we just update the CJS plugin?

Reply:
You could, but that would require updating Rollup 2 major versions forward, which is a pretty major change (everything broke when I tried).